tests/integration/hub_usage/test_hub_usage.py

```diff
@@ -76,3 +86,71 @@ def test_build_timeout_ready():
     HubIO(args).build()
     with Flow().add(uses=f'jinahub/pod.crafter.dummyhubexecutorslow:0.0.0-{jina_version}', timeout_ready = 20000):
         pass
+
+
+@pytest.mark.skipif(condition='os.environ.get(\'GITHUB_TOKEN\')==None', reason='Token not found')
+def test_hub_build_push(monkeypatch, access_token_github):
+    monkeypatch.setattr(Path, 'is_file', True)
+    monkeypatch.setattr(hubapi, '_fetch_access_token', access_token_github)
+    args = set_hub_build_parser().parse_args([str(cur_dir + '/hub-mwu'), '--push', '--host-info'])
+    summary = HubIO(args).build()
+
+    with open(cur_dir + '/hub-mwu' + '/manifest.yml') as fp:
+        manifest_jaml = JAML.load(fp, substitute=True)
+        manifest = expand_dict(manifest_jaml)
+
+    assert summary['is_build_success']
+    assert manifest['version'] == summary['version']
+    assert manifest['description'] == summary['manifest_info']['description']
+    assert manifest['author'] == summary['manifest_info']['author']
+    assert manifest['kind'] == summary['manifest_info']['kind']
+    assert manifest['type'] == summary['manifest_info']['type']
+    assert manifest['vendor'] == summary['manifest_info']['vendor']
+    assert manifest['keywords'] == summary['manifest_info']['keywords']
+
+    args = set_hub_list_parser().parse_args([
+        '--name', summary['manifest_info']['name'],
+        '--keywords', summary['manifest_info']['keywords'][0],
+        '--type', summary['manifest_info']['type']
+    ])
+    response = HubIO(args).list()
+    manifests = response
+
+    assert len(manifests) >= 1
+    assert manifests[0]['name'] == summary['manifest_info']['name']
+
+
```

Comment:
`@pytest.mark.skipif(condition='os.environ.get(\'GITHUB_TOKEN\')==None', reason='Token not found')` here too.